core/Cargo.toml

```diff
@@ -35,6 +35,8 @@ rlp_compress = { git = "https://github.com/CodeChain-io/rlp.git", version = "0.2
 rlp_derive = { git = "https://github.com/CodeChain-io/rlp.git", version = "0.2" }
 snap = "0.2"
 table = { path = "../util/table" }
+serde = "1.0"
+serde_derive = "1.0"
```

Comment:
We aren't using serde in core crate.
Since serializing a data structure can be different by the context, we don't provide a default serialization for the core data types.
Please create a new type in rpc/src/v1/types directory and create serialization code for the type.

Reply:
Which is preferred, to use serde remote derive (https://serde.rs/remote-derive.html), or to create a new type with the same name?

Reply:
Let's make it consistent with other types in the rpc trait. You can find them in the rpc/src/v1/types directory. Since they are creating a new type with the same name, let's use the method.